firecrown/parameters.py

```diff
@@ -63,16 +63,16 @@ class RequiredParameters:
     which implements lazy evaluation.
     """
 
-    def __init__(self, params_names: List[str]):
+    def __init__(self, params_names: Iterable[str]):
         """Construct an instance from a list of strings."""
```

Comment:
Comment should either say "from an Iterable yielding strings", or maybe just not describe the type (it is in the type annotations of the signature).

Reply:
Fixed in the last commit.